lib/shell.coffee

```diff
@@ -52,16 +51,39 @@ exports.getSubShellCommand = (command) ->
 # stdin is inherited from the parent process.
 #
 # @param {String} command - command
+# @param {Object} [options] - options
+# @param {String} [options.cwd] - current working directory
 # @returns {Promise}
 #
 # @example
 # shell.runCommand('echo hello').then ->
 # 	console.log('Done!')
 ###
-exports.runCommand = (command) ->
+exports.runCommand = (command, options = {}) ->
+	env = {}
+
+	if os.platform() is 'win32'
+
+		# Under Windows, openssh attempts to read SSH keys from
+		# `/home/<username>`, however this makes no sense in Windows.
+		# As a workaround, we can set the %HOME% environment variable
+		# to `/<home drive letter>/Users/<user>` to trick openssh
+		# to read ssh keys from `<home drive letter>:\Users\<user>\.ssh`
+		homedrive = _.get(process, 'env.homedrive', 'C:').slice(0, 1).toLowerCase()
+		homepath = _.get(process, 'env.homepath', '').replace(/\\/g, '/')
+		env.HOME = "/#{homedrive}#{homepath}"
+
 	subShellCommand = exports.getSubShellCommand(command)
 	spawn = child_process.spawn subShellCommand.program, subShellCommand.args,
 		stdio: 'inherit'
+		env: env
```

Comment:
@jviotti this is why it didn't work on my setup ^

These are the `ssh` binaries used before/after the PR:

Before
* `/usr/local/bin/ssh` - `OpenSSH_7.2p2, OpenSSL 1.0.2h  3 May 2016`

After
* `/usr/bin/ssh` - `OpenSSH_6.9p1, LibreSSL 2.1.8`